test/conftest.py

```diff
@@ -69,7 +69,7 @@ def pytest_collection_modifyitems(config, items):
 LEARNER_KWARGS = {
     ExperienceReplayLearner: {
         "memory_size": 30,
-        "memory_batch_size": 20,
+        "batch_memory_frac": 0.6,
```

Comment:
The ratios dont match: .6 * 30 != 20?

Reply:
Yes, will adjust this. But the tests don't depend on the exact numbers. All unit tests are passing. It's the integration tests that puzzle me.